src/Mold.cpp

```diff
@@ -0,0 +1,31 @@
+/** Copyright (C) 2017 Ultimaker - Released under terms of the AGPLv3 License */
+#include "Mold.h"
+#include "utils/intpoint.h"
+
+namespace cura
+{
+
+void Mold::process(Slicer& slicer, coord_t layer_height, double angle, coord_t width, coord_t open_polyline_width)
+{
+    Polygons mold_outline_above; // the outside of the mold
+
+    coord_t inset = tan(angle / 180 * M_PI) * layer_height;
```

Comment:
Ah no. Negative angles are possible if you want a really wide base.
Changed frontend such that the setting is limited to -89 to 90